src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -83,6 +77,12 @@ const AppCard: React.FC<AppCardProps> = ({
     [userPubkey, name, description, logo, url, setSubscribing, subscribed, notifyClientProxy]
   )
 
+  useEffect(() => {
+    if (subscribing && subscribed) {
+      showSuccessMessageToast(`Subscribed to ${name}`)
```

Comment:
@devceline Should we call the `setSubscribing(false)` in this block as well? Seems like we don't set it to false if not error occurs.